### ISSUE TITLE
Brave news settings bar missing when scrolling news after enabling Brave News

### ISSUE THREAD

**Issue by Uni-verse** [brave/brave-browser#21725]
## Description 
After enabling Brave News in the settings menu, the brave news menu bar is missing when scrolling through news.

Note: Relaunching Brave app will restore the Brave news settings bar

## Steps to reproduce 

   1. Clean install
   2. Navigate to menu > Brave News > Enable `Brave News`
   3. Enable `Show Brave News`
   4. Go back to homepage
   5. Scroll through Brave News

## Actual result 
Missing news settings bar

https://user-images.githubusercontent.com/17885425/158721622-21f307e7-a20d-4841-bc67-de21494b17ca.mp4

## Expected result
Show news settings bar

## Issue reproduces how often 
Easily, 100% repro rate

## Version/Channel Information:

- Can you reproduce this issue with the current Play Store version? 
- Can you reproduce this issue with the current Play Store Beta version? 
- Can you reproduce this issue with the current Play Store Nightly version?

## Device details
   - Install type (ARM, x86): `ARM`
   - Device type (Phone, Tablet, Phablet): `Samsung Galaxy S21`
   - Android version: `Android 12`
   
## Brave version
`Nightly 1.38.49, Chromium 99.04844.74`

### Website problems only
- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards?
- Is the issue reproducible on the latest version of Chrome?

**Comment by timchilds** [comment]
@Uni-verse I cannot reproduce this issue on my device (Samsung Galaxy A51 / Android 11).

cc @anthonypkeane @bsclifton

**Comment by kjozwiak** [comment]
@tapanmodh technically this should still be in the `1.39.x` milestone as the original PR landed in `1.39.x` and this wasn't uplifted yet. Once https://github.com/brave/brave-core/pull/12931 is merged into `1.38.x`, the release team will move the issue into the correct milestone, in this case, `1.38.x`. You can view/read the process via https://github.com/brave/brave-browser/wiki/Triage-Guidelines#uplifting.

**Comment by kjozwiak** [comment]
https://github.com/brave/brave-core/pull/12817 was verified on Nightly as per https://github.com/brave/brave-core/pull/12817#issuecomment-1092426388 and uplifted via https://github.com/brave/brave-core/pull/12931.

Above requires `1.38.86` or higher for `1.38.x` verification.

**Comment by Uni-verse** [comment]
Verified using `1.38.99, Chromium 100.0.4896.127`

Devices: 
Samsung Galaxy S21 | Android 12
Samsung Galaxy Tab S7 | Android 12

- Verified brave news settings bar is present when scrolling news after enabling Brave News from settings
- Verified brave setting bar is functional

https://user-images.githubusercontent.com/17885425/164115928-f2b47fa4-3402-407b-b839-fb17f411d07d.mp4
